### ISSUE TITLE
Earlier creation of image files.

### ISSUE THREAD

**Issue by spikefishjohn** [GNS3/gns3-gui#3052]
**Is your feature request related to a problem? Please describe.**
I've noticed that GNS3 doesn't create the images files for, at least qemu imges until you start the VM. I've tested this with 2.2.7 with the GUI and API. The issue I have is I want to create the image, then use guest fish to modify the image before starting the image. As it stands we have to issue a start followed by a stop right after that to get the files for the device to deploy.

**Describe the solution you'd like**
I just started looking through code trying to figure out when qemu drives are deployed, but I'm assuming there is a reason for delay until start so maybe having an API option for create to create files right now? Not sure how hard that is.

**Describe alternatives you've considered**
quick start / stop to get files to deploy. 

**Additional context**

**Comment by grossmj** [comment]
Duplicate of https://github.com/GNS3/gns3-gui/issues/3053
